oracle/provider/binance.go

```diff
@@ -239,7 +239,7 @@ func (p *BinanceProvider) getCandlePrices(key string) ([]types.CandlePrice, erro
 	return candleList, nil
 }
 
-func (p *BinanceProvider) messageReceived(messageType int, bz []byte) {
+func (p *BinanceProvider) messageReceived(_ int, bz []byte) {
```

Comment:
I can't remember now, why do we need to check the message type for some providers and not others?

Reply:
The providers that we don't check the message type only ever send one type (either always binary or always string). It would probably be a good idea to check in each one like we do here: https://github.com/ojo-network/price-feeder/blob/8e42ab144172ed326b474a71bca802b048db2d56/oracle/provider/huobi.go#L210 in case the provider decides to start sending new messages.

Reply:
Added a note in the price feeder v1 epic